src/osmf/js/osmf_flash.js

```diff
@@ -590,8 +590,8 @@ require("../../../html5-common/js/utils/constants.js");
                                "seekRange": { "begin": seekRange_start, "end": seekRange_end } });
     };
 
-    var raiseCanPlayThrough = function() {
-      newController.notify(newController.EVENTS.BUFFERED);
+    var raiseCanPlayThrough = function(event) {
```

Comment:
Can we add a JS doc for this? I understand there wasn't one before, but this seems like a good time to add one :)